### PR TITLE
Added missing periods to ERR resources

### LINKED ISSUE #622: C# error messages are sentences. Most of them end with a period. Some of them are missing the period. We should be consistent.

### PR DESCRIPTION
**Pull request by khellang**
Fixes #622

Cue discussion!

### CODE REVIEW

File: src/Compilers/VisualBasic/Portable/VBResources.Designer.vb [hunk 1]
```diff
@@ -170,7 +170,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
         
         '''<summary>
-        '''  Looks up a localized string similar to Cannot find the interop type that matches the embedded type &apos;{0}&apos;. Are you missing an assembly reference?.
+        '''  Looks up a localized string similar to Cannot find the interop type that matches the embedded type &apos;{0}&apos;. Are you missing an assembly reference?..
```
Comment (khellang) [inline src/Compilers/VisualBasic/Portable/VBResources.Designer.vb]: It looks a bit weird when ResGen adds that extra period :disappointed:

**Comment by khellang** [comment]
I'm sure there are some periods that looks weird and should be removed again though.

**Comment by khellang** [comment]
Ouch. Looks like there's quite a bit of asserts on these messages. I'll clean up the tests (and most obvious errors) later today.

**Comment by gafter** [comment]
This failed the merge and test. You are welcome to resubmit the pull request if/when it passes tests.

**Pull request closed without merging**